faq.html

```diff
@@ -62,8 +69,8 @@ <h4 class="panel-title">
           <div id="G-1" class="panel-collapse collapse">
             <div class="panel-body">
               <ol>
-                <li>Uninstall Prestissimo:<br />On Android 6.0+ (Marshmallow and newer), you can use Android's native mediaplayer for variable playback speed. <br />On older devices, enable <i>Sonic mediaplayer</i>.</li>
-                <li>Prefer MP3 feeds over other formats, especially MP4/AAC:<br />We use Android's built-in audio decoders. A lot of devices are shipped with rather buggy MP4 decoders that lead to various playback issues and even native crashes (think <i>Bluescreen</i>).<br />
+                <li>Uninstall Prestissimo:<br>On Android 6.0+ (Marshmallow and newer), you can use Android's native mediaplayer for variable playback speed. <br>On older devices, enable <i>Sonic mediaplayer</i>.</li>
+                <li>Prefer MP3 feeds over other formats, especially MP4/AAC:<br>We use Android's built-in audio decoders. A lot of devices are shipped with rather buggy MP4 decoders that lead to various playback issues and even native crashes (think <i>Bluescreen</i>).<br>
```

Comment:
What's the reason for changing the `<br />` tags? I think they are fine as they are :) Fewer code changes make the git history easier to read.

Reply:
The files are html5 which uses `<br>`. The `<br />`  is allowed in html5 but should not be used because it is xhtml style.

Reply:
I still don't really like the change (the html5 spec explicitly allows both and I think we should stay consistent instead of having another style in each file). Anyway, I'm not too involved in the website maintenance, so I can live with your change :)